lib/rules/no-identical-tests.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @fileoverview disallow identical tests
+ * @author 薛定谔的猫<hh_2013@foxmail.com>
+ */
+
+'use strict';
+
+const utils = require('../utils');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'disallow identical tests',
+      category: 'Tests',
+      recommended: true,
+    },
+    fixable: null,  // or "code" or "whitespace"
+    schema: [],
+  },
+
+  create (context) {
+    // ----------------------------------------------------------------------
+    // Public
+    // ----------------------------------------------------------------------
+    const message = 'This test case should not be identical to someone else.';
```

Comment:
Nitpick: Can you reword this message to something like this? "This test case is identical to another case."